pkg/agent/run.go

```diff
@@ -97,6 +98,11 @@ func run(ctx context.Context, cfg cmds.Agent, proxy proxy.Proxy) error {
 	}
 
 	if !nodeConfig.NoFlannel {
+		if (nodeConfig.FlannelExternalIP) && (len(nodeConfig.AgentConfig.NodeExternalIPs) == 0) {
+			logrus.Warnf("Flannel-external-ip flag requires node-external-ip to be set.")
+		} else if (nodeConfig.FlannelBackend != types.FlannelBackendWireguardNative) && (nodeConfig.FlannelBackend != types.FlannelBackendIPSEC) {
+			logrus.Warnf("Unencrypted flannel traffic is sent using externalIPs. Please consider using an encrypted flannel backend")
```

Comment:
How about this? Hoping that printing the current backend will help folks find the flannel backend docs and choose a different one.
```suggestion
			logrus.Warnf("Flannel is using external addresses with an insecure backend: %v. Please consider using an encrypting flannel backend.", nodeConfig.FlannelBackend)
```

Reply:
good idea